apps/etno/.env.example

```
@@ -72,3 +72,12 @@ OPENSEARCH_USERNAME=
 OPENSEARCH_PASSWORD=
 
 FRONTEND_URL=http://localhost:3000
+
+PROMETHEUS_ENABLED=true
+PROMETHEUS_ROUTE=metrics
+PROMETHEUS_NAMESPACE=app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROMETHEUS_NAMESPACE key should go before the PROMETHEUS_ROUTE key

(UnorderedKey)